### PR TITLE
Display show details

### PR DESCRIPTION
**Pull request by codexbella**

### CODE REVIEW

File: backend/src/main/java/de/codexbella/ContentController.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 import java.security.Principal;
 import java.util.List;
+import java.util.NoSuchElementException;
```
Comment (gossie) [inline backend/src/main/java/de/codexbella/ContentController.java:13]: Why was this import added?

File: backend/src/main/java/de/codexbella/ContentService.java [hunk 2]
```diff
@@ -11,6 +11,7 @@
 import org.springframework.web.client.RestTemplate;
 
 import java.util.List;
+import java.util.NoSuchElementException;
```
Comment (gossie) [inline backend/src/main/java/de/codexbella/ContentService.java:14]: Why was this import added?

**Review by gossie — APPROVED** [review]
Just two questions.

File: backend/src/main/java/de/codexbella/ContentController.java (hunk 1 above)
Comment (codexbella) [reply backend/src/main/java/de/codexbella/ContentController.java:13]: Good catch - that was before `ResponseEntity.of` with the `Optional<Show>`.

File: backend/src/main/java/de/codexbella/ContentService.java (hunk 2 above)
Comment (codexbella) [reply backend/src/main/java/de/codexbella/ContentService.java:14]: Same.
